docs/index.md

```diff
@@ -737,15 +737,15 @@ $ mocha
 
 Many reporters will display test duration and flag tests that are slow (default: 75ms), as shown here with the "spec" reporter:
 
-![test duration](images/reporter-spec-duration.png?withoutEnlargement&resize=920,9999){:class="screenshot"}
+![test duration](images/reporter-spec-duration.png?withoutEnlargement&resize=920,9999){:class="screenshot" lazyload="on"}
```

Comment:
Could the `lazyload` have been set in CSS for `class="screenshot"` instead?

Reply:
Couple lines of JS isn't it? I used to have something similar that added `rel="external"` to all `<a>` elements with nonlocal hrefs.

Reply:
[caniuse-lazyload](https://caniuse.com/#feat=lazyload)
[Blink lazyload](https://www.bleepingcomputer.com/news/google/built-in-lazy-loading-lands-in-google-chrome-canary/)

Too bad the vendors didn't read the same spec...